examples/permsapp.py

```diff
@@ -0,0 +1,170 @@
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+"""Minimal Flask application example with access control enabled.
+
+Run example development server:
+
+.. code-block:: console
+    $ cd examples
+    $ flask -a permsapp.py db init
+    $ flask -a permsapp.py db create
+    $ flask -a permsapp.py fixtures records
+    $ flask -a permsapp.py fixtures access
+    $ flask -a permsapp.py --debug run
+
+
+Try to get record 1:
+
+    $ curl -XGET http://localhost:5000/records/1
+
+Login as admin@invenio-software.org (password 123456) and get record 1:
+
+    $ curl -XPOST http://localhost:5000/login -d 'email=admin%40invenio-software.org&password=123456' -c mycookie
+    $ curl -XGET http://localhost:5000/records/1 -b mycookie
+
+Login as forbidden@invenio-software.org (password 123456), hwo has not
+the permission to read record 1, and try to get record 1:
+
+    $ curl -XPOST http://localhost:5000/login -d 'email=forbidden%40invenio-software.org&password=123456' -c mycookie2
+    $ curl -XGET http://localhost:5000/records/1 -b mycookie2
+"""  # noqa
+
+from __future__ import absolute_import, print_function
+
+import os
+
+from flask import Flask
+from flask_celeryext import FlaskCeleryExt
+from flask_cli import FlaskCLI
+from flask_menu import Menu
+from flask_security.utils import encrypt_password
+from invenio_accounts import InvenioAccounts
+from invenio_accounts.views import blueprint
+from invenio_db import InvenioDB, db
+from invenio_pidstore import InvenioPIDStore
+from invenio_records import InvenioRecords
+from invenio_records.permissions import records_create_all, \
+    records_delete_all, records_read_all, records_update_all
+from invenio_rest import InvenioREST
+
+from invenio_access import InvenioAccess
+from invenio_access.models import ActionUsers
+from invenio_records_rest import InvenioRecordsREST
+
+
+# create application's instance directory. Needed for this example only.
+current_dir = os.path.dirname(os.path.realpath(__file__))
+instance_dir = os.path.join(current_dir, 'permsapp')
+if not os.path.exists(instance_dir):
+    os.makedirs(instance_dir)
+
+# Create Flask application
+app = Flask(__name__, instance_path=instance_dir)
+app.config.update(
+    CELERY_ALWAYS_EAGER=True,
+    CELERY_CACHE_BACKEND="memory",
+    CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
+    CELERY_RESULT_BACKEND="cache",
+    # Install Principal and Login extensions
+    WTF_CSRF_ENABLED=False,
+    ACCOUNTS_USE_CELERY=False,
+    SECRET_KEY='CHANGE_ME',
+    SECURITY_PASSWORD_SALT='CHANGE_ME_ALSO',
+)
+FlaskCLI(app)
+FlaskCeleryExt(app)
+InvenioDB(app)
+InvenioREST(app)
+InvenioPIDStore(app)
+InvenioRecords(app)
+Menu(app)
+accounts = InvenioAccounts(app)
+app.register_blueprint(blueprint)
+InvenioAccess(app)
+InvenioRecordsREST(app)
+
+rec_uuid = '9107e6ef-fea4-4971-ae2f-934d2fdcaa30'
```

Comment:
Probably we should use the same uuid as in records-ui:
``deadbeef-1234-5678-ba11-b100dc0ffee5``